include/deal.II/grid/tria_accessor.h

```diff
@@ -1238,6 +1238,17 @@ class TriaAccessor : public TriaAccessorBase<structdim, dim, spacedim>
   double diameter () const;
 
   /**
+   * Return a pair of the center and the diameter of the smallest enclosing
+   * sphere of the object.
+   *
+   * The diameter of the smallest enclosing sphere is the diameter of
+   * the object (i.e., the largest diagonal).
+   * The center of the smallest enclosing sphere is the mid point
+   * of the vertices of the largest diagonal of the object.
+   */
```

Comment:
If this is true, the algorithm might be correct. Then, we should add some `Assert` to make sure the cell in question actually is a parallelotope.